src/cjson/lua_cjson.c

```diff
@@ -57,17 +59,14 @@
 #endif
 
 #ifndef CJSON_VERSION
-#define CJSON_VERSION   "2.1.0.9"
+#define CJSON_VERSION   "2.1.0.11"
```

Comment:
current is 2.1.0.14, but apparently they haven't bumped the version in this file. (Release seems to be a bit haphazard...)